package/kedro_viz/server.py

```diff
@@ -36,11 +36,9 @@ def populate_data(
         session_class = make_db_session_factory(session_store.location)
         data_access_manager.set_db_session(session_class)
 
-    # resolve the dataset factory patterns
-    data_access_manager.resolve_dataset_factory_patterns(catalog, pipelines)
-
-    # add catalog and relevant tracking datasets
-    data_access_manager.add_catalog(catalog)
+    # resolve dataset factory patterns, add catalog
```

Comment:
we can remove this comment.

Reply:
okay, but add_catalog method does all the 3 functions right ?

Reply:
you want me to remove entire comment as we have doc strings ?